### ISSUE TITLE
Some uninstallation scripts that use rm fail

### ISSUE THREAD

**Issue by danstewart** [folkswithhats/fedy#442]
When uninstalling programs that try to delete from certain directories I get a popup asking me to confirm it's ok to delete but it fails anyway and outputs no errors.

Popup: https://imgur.com/a/ZpSU0
Failure and (lack of) console output: https://imgur.com/a/XRlPM

List of plugins that fail:
- Handbrake
- Arc theme
- Adapta theme (not yet added)

Handbrake and Arc both delete from `/etc/yum.repos.d/` and Adapta deletes from `/usr/share/themes/`

**Comment by danstewart** [comment]
JS code that handles the popup:
https://github.com/folkswithhats/fedy/blob/master/app.js#L302

```javascript
        if (malicious) {
            this._showDialog({
                type: "question",
                text: "The plugin <b>" + GLib.markup_escape_text(plugin.label, -1) + "</b> is trying to run the command \n" +
                      "<tt>" + GLib.markup_escape_text(command, -1) + "</tt>, \n" +
                      "which might <b>" + GLib.markup_escape_text(description, -1) + "</b>. \n" +
                      "Continue anyways?"
            }, (dialog, response) => {
                switch (response) {
                case Gtk.ResponseType.OK:
                    runner.call(this, plugin.path, cmd, cb);
                    break;
                default:
                    cb(null, 1);
                    break;
                }
            });

            return;
}
```

**Comment by danstewart** [comment]
Clicking "Yes" hits the default in the switch (Hit 2).
`print(response)` outputs `-8`

```javascript
            }, (dialog, response) => {
                print(response);
                switch (response) {
                case Gtk.ResponseType.OK:
                    print("Hit 1"); 
                    runner.call(this, plugin.path, cmd, cb);
                    break;
                default:
                    print("Hit 2"); 
                    cb(null, 1);
                    break;
                }
            });
```

**Comment by satya164** [comment]
@danstewart looks like there's a typo for response type. It should be `Gtk.ResponseType.YES` I think. Can you change it and see if works?

There seems to be another bug with the regex which flags this command incorrectly, but we'll get to that.

**Comment by danstewart** [comment]
@satya164 That's so obvious now you say it. Worked perfectly. I'll send a PR. Thanks.

What's the regex issue?

**Comment by satya164** [comment]
@danstewart nah, the regex is fine, my mistake :)

**Comment by danstewart** [comment]
@satya164 No bother, thanks for the help!

**Comment by danstewart** [comment]
Fixed by #443.
